src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/RoboticsAddressMapper.java

```diff
@@ -18,14 +22,18 @@
 
     public RoboticsAddress toRoboticsAddress(Address address) {
         requireNonNull(address);
-        return RoboticsAddress.builder()
-            .addressLine1(address.firstNonNull())
-            .addressLine2(Optional.ofNullable(address.secondNonNull()).orElse("-"))
-            .addressLine3(address.thirdNonNull())
-            .addressLine4(address.fourthNonNull())
-            .addressLine5(address.fifthNonNull())
-            .postCode(address.getPostCode())
-            .build();
+
+        if ((address.getAddressLine1() != null && address.getAddressLine1().length() > 35)
+            || (address.getAddressLine2() != null && address.getAddressLine2().length() > 35)) {
+            try {
+                Address alteredAddress = tryToRollOverAddressLines(address);
+                return toNonDefaultRoboticsAddress(alteredAddress);
```

Comment:
Hey, do we a `toNonDefaultRoboticsAddress` and a `toDefaultRoboticsAddress`? I am not sure I understand the reason why it's there...